prov/verbs/src/fi_verbs.c

```diff
@@ -990,14 +1004,8 @@ fi_ibv_create_ep(const char *node, const char *service,
 
 	ret = rdma_create_ep(id, _rai, NULL, NULL);
 	if (ret) {
+		VERBS_INFO_ENO(FABRIC, "rdma_create_ep", errno);
 		ret = -errno;
-		if (ret == -ENOENT) {
-			FI_WARN(&fi_ibv_prov, FI_LOG_CORE,
-				"rdma_create_ep()-->ENOENT; "
-				"likely usnic bug, skipping verbs provider.\n");
-			ret = -FI_ENODATA;
-		}
-		FI_INFO(&fi_ibv_prov, FI_LOG_CORE, "Unable to create rdma_cm id\n");
```

Comment:
The filtering logic wouldn't help but since in the active endpoint case, the app passed a specific address to bind to, which would be the address of a valid verbs device, I was of the idea that rdma_create_ep wouldn't take usNIC into consideration. Again, this is all a guess. Let me go thru with this PR as-is. Thank you for taking a look at this.